svg/camera.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
```

Comment:
Most of the icons in this repo has `8` for width and height`<svg xmlns="http://www.w3.org/2000/svg" width="8" height="8" viewBox="0 0 8 8">` 
do you think this can be a problem? i guess not, but i'm not sure.

Reply:
I found out some time ago that most of the repos seems to have different values for their icons so no problem with that so far.